database/sqlite3.ksy

```diff
@@ -32,213 +32,431 @@ doc: |
   (`root_page`) is always "btree_page".
```

Comment:
`root_page` is still referenced in the docstring

Reply:
paragraph updated